src/bin/ion/commands/beta/inspect.rs

```diff
@@ -6,163 +6,153 @@ use std::io::BufWriter;
+    }
+
+    fn configure_args(&self, command: Command) -> Command {
+        command
+            .with_input()
+            .with_output()
+            .arg(
+                // This is named `skip-bytes` instead of `skip` to accommodate a future `skip-values` option.
+                Arg::new("skip-bytes")
+                    .long("skip-bytes")
+                    .short('s')
+                    .default_value("0")
+                    .hide_default_value(true)
+                    .help("Do not display any user values for the first `n` bytes of Ion data.")
+                    .long_help(
+                        "When specified, the inspector will skip ahead `n` bytes before
 beginning to display the contents of the stream. System values like
 Ion version markers and symbol tables in the bytes being skipped will
 still be displayed. If the requested number of bytes falls in the
 middle of a value, the whole value (complete with field ID and
 annotations if applicable) will be displayed. If the value is nested
 in one or more containers, those containers will be displayed too.",
-                ),
-        )
-        .arg(
-            // This is named `limit-bytes` instead of `limit` to accommodate a future `limit-values` option.
-            Arg::new("limit-bytes")
-                .long("limit-bytes")
-                .short('l')
-                .default_value("0")
-                .hide_default_value(true)
-                .help("Only display the next 'n' bytes of Ion data.")
-                .long_help(
-                    "When specified, the inspector will stop printing values after
+                    ),
+            )
+            .arg(
+                // This is named `limit-bytes` instead of `limit` to accommodate a future `limit-values` option.
+                Arg::new("limit-bytes")
+                    .long("limit-bytes")
+                    .short('l')
+                    .default_value("0")
+                    .hide_default_value(true)
+                    .help("Only display the next 'n' bytes of Ion data.")
+                    .long_help(
+                        "When specified, the inspector will stop printing values after
 processing `n` bytes of Ion data. If `n` falls within a value, the
 complete value will be displayed.",
-                ),
-        )
-}
-
-// Create a type alias to simplify working with a shared reference to our output stream.
-type OutputRef = Box<dyn io::Write>;
-// * The output stream could be STDOUT or a file handle, so we use `dyn io::Write` to abstract
-//   over the two implementations.
-// * The Drop implementation will ensure that the output stream is flushed when the last reference
-//   is dropped, so we don't need to do that manually.
+                    ),
+            )
+    }
 
-// This function is invoked by the `inspect` command's parent, `beta`.
-pub fn run(_command_name: &str, matches: &ArgMatches) -> Result<()> {
```

Comment:
🗺️ There are no logic changes within this method, just lots of moves and indentation changes.